tests/blocks/picacoin_mainnet_302

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
you added both `tests/blocks/picacoin_mainnet_302` and `tests/blocks/picacoin_mainnet_302.json`.
I haven't checked the contents of these file,
but in terms of names: rm the first, only keep the json file.